packages/docs/pages/reference/elements/md.mdx

```diff
@@ -7,7 +7,7 @@ import {
 
 # `<md>`
 
-Short description of what the `md` element does.
+Used to produce multiline displays of mathematics. Individual lines of the display are entered as the content of `mrow` tags. The displayed expression can be aligned in columns using `\amp` within each individual `mrow`.  
```

Comment:
Put angles around references to tags. E.g., write
```
`<mrow>`
```
instead of
```
`mrow`
```

Reply:
Should mention that you can use `\amp` or `&amp;`; author's choice :-)

Reply:
How's this?